CRITICAL - 2023-06-13 00:56:51 --> Undefined array key "activo"
in APPPATH\Views\usuariosRegistrados.php on line 33.
 1 APPPATH\Views\usuariosRegistrados.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "activo"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\usuariosRegistrados.php', 33)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\usuariosRegistrados.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('usuariosRegistrados', [], true)
 5 APPPATH\Controllers\UserController.php(19): view('usuariosRegistrados', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UserController->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 00:57:26 --> Undefined array key "activo"
in APPPATH\Views\usuariosRegistrados.php on line 33.
 1 APPPATH\Views\usuariosRegistrados.php(33): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "activo"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\usuariosRegistrados.php', 33)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\usuariosRegistrados.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('usuariosRegistrados', [], true)
 5 APPPATH\Controllers\UserController.php(19): view('usuariosRegistrados', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UserController->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 00:59:14 --> mysqli_sql_exception: Table 'db_yunes.consultas' doesn't exist in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\Yunes\app\Controllers\ConsultasController.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\ConsultasController->cargarConsultas()
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-06-13 00:59:14 --> Table 'db_yunes.consultas' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `consultas`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `consultas`')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `consultas`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `consultas`', [], false)
 5 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\ConsultasController.php(20): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->cargarConsultas()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 01:16:46 --> Invalid file: "ventasRealizadas.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('ventasRealizadas.php')
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('ventasRealizadas', [], true)
 3 APPPATH\Controllers\VentasController.php(38): view('ventasRealizadas', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 01:17:11 --> Undefined array key "id_usuario"
in APPPATH\Views\ventas.php on line 22.
 1 APPPATH\Views\ventas.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_usuario"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php', 22)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('ventas', [], true)
 5 APPPATH\Controllers\VentasController.php(38): view('ventas', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 01:18:05 --> Undefined array key "id_usuario"
in APPPATH\Views\ventas.php on line 29.
 1 APPPATH\Views\ventas.php(29): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_usuario"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php', 29)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('ventas', [], true)
 5 APPPATH\Controllers\VentasController.php(38): view('ventas', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 01:20:48 --> Undefined array key "id_usuario"
in APPPATH\Views\ventas.php on line 29.
 1 APPPATH\Views\ventas.php(29): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_usuario"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php', 29)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('ventas', [], true)
 5 APPPATH\Controllers\VentasController.php(38): view('ventas', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 01:20:49 --> Undefined array key "id_usuario"
in APPPATH\Views\ventas.php on line 29.
 1 APPPATH\Views\ventas.php(29): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_usuario"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php', 29)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('ventas', [], true)
 5 APPPATH\Controllers\VentasController.php(38): view('ventas', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 01:21:30 --> Undefined array key "Total"
in APPPATH\Views\ventas.php on line 38.
 1 APPPATH\Views\ventas.php(38): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "Total"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php', 38)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('ventas', [], true)
 5 APPPATH\Controllers\VentasController.php(38): view('ventas', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 01:52:46 --> Undefined variable $usuarioValue
in APPPATH\Views\ventas.php on line 19.
 1 APPPATH\Views\ventas.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $usuarioValue', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php', 19)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('ventas', [], true)
 5 APPPATH\Controllers\VentasController.php(38): view('ventas', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 01:56:39 --> Undefined array key "id"
in APPPATH\Views\ventas.php on line 16.
 1 APPPATH\Views\ventas.php(16): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php', 16)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('ventas', [], true)
 5 APPPATH\Controllers\VentasController.php(38): view('ventas', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 01:56:41 --> Undefined array key "id"
in APPPATH\Views\ventas.php on line 16.
 1 APPPATH\Views\ventas.php(16): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php', 16)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('ventas', [], true)
 5 APPPATH\Controllers\VentasController.php(38): view('ventas', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 01:57:19 --> Undefined array key "nombre"
in APPPATH\Views\ventas.php on line 16.
 1 APPPATH\Views\ventas.php(16): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nombre"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php', 16)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('ventas', [], true)
 5 APPPATH\Controllers\VentasController.php(38): view('ventas', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 02:09:59 --> Undefined variable $usuarioValue
in APPPATH\Views\ventas.php on line 22.
 1 APPPATH\Views\ventas.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $usuarioValue', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php', 22)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\ventas.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('ventas', [], true)
 5 APPPATH\Controllers\VentasController.php(38): view('ventas', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 02:14:32 --> Undefined array key "Total"
in APPPATH\Views\detalleFactura.php on line 28.
 1 APPPATH\Views\detalleFactura.php(28): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "Total"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php', 28)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('detalleFactura', [], true)
 5 APPPATH\Controllers\VentasController.php(55): view('detalleFactura', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->detalleFactura('33')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 02:18:31 --> Undefined array key "name"
in APPPATH\Views\detalleFactura.php on line 18.
 1 APPPATH\Views\detalleFactura.php(18): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "name"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php', 18)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('detalleFactura', [], true)
 5 APPPATH\Controllers\VentasController.php(55): view('detalleFactura', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->detalleFactura('33')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 02:31:52 --> Undefined array key "subtotal"
in APPPATH\Views\detalleFactura.php on line 22.
 1 APPPATH\Views\detalleFactura.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "subtotal"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php', 22)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('detalleFactura', [], true)
 5 APPPATH\Controllers\VentasController.php(55): view('detalleFactura', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->detalleFactura('33')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 02:31:57 --> Undefined array key "qty"
in APPPATH\Views\detalleFactura.php on line 20.
 1 APPPATH\Views\detalleFactura.php(20): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "qty"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php', 20)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('detalleFactura', [], true)
 5 APPPATH\Controllers\VentasController.php(55): view('detalleFactura', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->detalleFactura('33')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 02:35:49 --> Undefined array key "descripcion"
in APPPATH\Views\detalleFactura.php on line 19.
 1 APPPATH\Views\detalleFactura.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "descripcion"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php', 19)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('detalleFactura', [], true)
 5 APPPATH\Controllers\VentasController.php(55): view('detalleFactura', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->detalleFactura('33')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 02:35:57 --> Undefined array key "qty"
in APPPATH\Views\detalleFactura.php on line 20.
 1 APPPATH\Views\detalleFactura.php(20): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "qty"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php', 20)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('detalleFactura', [], true)
 5 APPPATH\Controllers\VentasController.php(55): view('detalleFactura', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->detalleFactura('33')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 02:46:59 --> Undefined array key "qty"
in APPPATH\Views\detalleFactura.php on line 20.
 1 APPPATH\Views\detalleFactura.php(20): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "qty"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php', 20)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('detalleFactura', [], true)
 5 APPPATH\Controllers\VentasController.php(55): view('detalleFactura', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->detalleFactura('33')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 02:47:00 --> Undefined array key "qty"
in APPPATH\Views\detalleFactura.php on line 20.
 1 APPPATH\Views\detalleFactura.php(20): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "qty"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php', 20)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\detalleFactura.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('detalleFactura', [], true)
 5 APPPATH\Controllers\VentasController.php(55): view('detalleFactura', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->detalleFactura('33')
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 03:21:20 --> Undefined array key "producto_id"
in APPPATH\Controllers\VentasController.php on line 49.
 1 APPPATH\Controllers\VentasController.php(49): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "producto_id"', 'C:\\xampp\\htdocs\\Yunes\\app\\Controllers\\VentasController.php', 49)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->detalleFactura('24')
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 03:21:36 --> Array to string conversion
in SYSTEMPATH\Database\Query.php on line 329.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'C:\\xampp\\htdocs\\Yunes\\system\\Database\\Query.php', 329)
 2 SYSTEMPATH\Database\Query.php(329): implode(',', [...])
 3 SYSTEMPATH\Database\Query.php(310): CodeIgniter\Database\Query->matchNamedBinds('SELECT *
FROM `productos`
WHERE `id` = :id:', [...])
 4 SYSTEMPATH\Database\Query.php(159): CodeIgniter\Database\Query->compileBinds()
 5 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\Query->getQuery()
 6 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `productos`
WHERE `id` = :id:', [...], false)
 7 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 8 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 9 APPPATH\Controllers\VentasController.php(49): CodeIgniter\BaseModel->findAll()
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->detalleFactura('24')
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 03:21:43 --> Array to string conversion
in SYSTEMPATH\Database\Query.php on line 329.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'C:\\xampp\\htdocs\\Yunes\\system\\Database\\Query.php', 329)
 2 SYSTEMPATH\Database\Query.php(329): implode(',', [...])
 3 SYSTEMPATH\Database\Query.php(310): CodeIgniter\Database\Query->matchNamedBinds('SELECT *
FROM `productos`
WHERE `id` = :id:', [...])
 4 SYSTEMPATH\Database\Query.php(159): CodeIgniter\Database\Query->compileBinds()
 5 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\Query->getQuery()
 6 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `productos`
WHERE `id` = :id:', [...], false)
 7 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 8 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 9 APPPATH\Controllers\VentasController.php(49): CodeIgniter\BaseModel->findAll()
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->detalleFactura('24')
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 03:29:25 --> mysqli_sql_exception: Unknown column 'email' in 'field list' in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 03:29:25 --> Unknown column 'email' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (\'nicolas\', \'yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (\'nicolas\', \'yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (\'nicolas\', \'yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (:nombre:, :apellido:, :email:, :mensaje:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 03:29:37 --> mysqli_sql_exception: Unknown column 'email' in 'field list' in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 03:29:37 --> Unknown column 'email' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (\'nicolas\', \'yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (\'nicolas\', \'yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (\'nicolas\', \'yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (:nombre:, :apellido:, :email:, :mensaje:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 03:41:10 --> mysqli_sql_exception: Unknown column 'email' in 'field list' in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 03:41:10 --> Unknown column 'email' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (\'nicolas\', \'yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (\'nicolas\', \'yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (\'nicolas\', \'yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (:nombre:, :apellido:, :email:, :mensaje:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 03:53:16 --> mysqli_sql_exception: Unknown column 'email' in 'field list' in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 03:53:16 --> Unknown column 'email' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (\'nicolas\', \'yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (\'nicolas\', \'yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (\'nicolas\', \'yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `consultas` (`nombre`, `apellido`, `email`, `mensaje`) VALUES (:nombre:, :apellido:, :email:, :mensaje:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 04:24:08 --> mysqli_sql_exception: Unknown column 'email' in 'field list' in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 04:24:08 --> Unknown column 'email' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'nicolas yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'nicolas yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'nicolas yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (:nombre:, :email:, :mensaje:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 04:24:48 --> mysqli_sql_exception: Unknown column 'email' in 'field list' in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 04:24:48 --> Unknown column 'email' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'nicolas yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'nicolas yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'nicolas yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (:nombre:, :email:, :mensaje:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 04:24:52 --> mysqli_sql_exception: Unknown column 'email' in 'field list' in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 04:24:52 --> Unknown column 'email' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'nicolas yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'nicolas yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'nicolas yunes\', \'nicolaseduardoyunes@gmail.com\', \'123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (:nombre:, :email:, :mensaje:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 04:26:22 --> mysqli_sql_exception: Unknown column 'email' in 'field list' in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 04:26:22 --> Unknown column 'email' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'nicolas yunes\', \'nicolaseduardoyunes@gmail.com\', \'123123\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'nicolas yunes\', \'nicolaseduardoyunes@gmail.com\', \'123123\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'nicolas yunes\', \'nicolaseduardoyunes@gmail.com\', \'123123\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (:nombre:, :email:, :mensaje:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 04:26:47 --> mysqli_sql_exception: Unknown column 'email' in 'field list' in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 04:26:47 --> Unknown column 'email' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'clienteX\', \'cl@cl\', \'1234234234234\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'clienteX\', \'cl@cl\', \'1234234234234\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'clienteX\', \'cl@cl\', \'1234234234234\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (:nombre:, :email:, :mensaje:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 04:27:31 --> mysqli_sql_exception: Unknown column 'email' in 'field list' in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 04:27:31 --> Unknown column 'email' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'clienteX\', \'cl@cl\', \'1234234234234\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'clienteX\', \'cl@cl\', \'1234234234234\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'clienteX\', \'cl@cl\', \'1234234234234\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (:nombre:, :email:, :mensaje:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 04:30:05 --> mysqli_sql_exception: Unknown column 'email' in 'field list' in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 04:30:05 --> Unknown column 'email' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'clienteX\', \'cl@cl\', \'1234234234234\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'clienteX\', \'cl@cl\', \'1234234234234\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'clienteX\', \'cl@cl\', \'1234234234234\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (:nombre:, :email:, :mensaje:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 04:30:51 --> mysqli_sql_exception: Unknown column 'email' in 'field list' in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 04:30:51 --> Unknown column 'email' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'clienteX\', \'cl@cl\', \'1234234234234\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'clienteX\', \'cl@cl\', \'1234234234234\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'clienteX\', \'cl@cl\', \'1234234234234\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (:nombre:, :email:, :mensaje:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 04:30:54 --> mysqli_sql_exception: Unknown column 'email' in 'field list' in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `co...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `co...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `co...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `co...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 04:30:54 --> Unknown column 'email' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'clienteX\', \'cl@cl\', \'1234234234234\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'clienteX\', \'cl@cl\', \'1234234234234\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (\'clienteX\', \'cl@cl\', \'1234234234234\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `consultas` (`nombre`, `email`, `mensaje`) VALUES (:nombre:, :email:, :mensaje:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ConsultasController.php(72): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->agregarConsulta()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 11:58:17 --> Undefined array key "qty"
in APPPATH\Views\catalogo.php on line 37.
 1 APPPATH\Views\catalogo.php(37): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "qty"', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\catalogo.php', 37)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\catalogo.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('catalogo', [], true)
 5 APPPATH\Controllers\ProductController.php(26): view('catalogo', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProductController->catalogo()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 09:00:20 --> mysqli_sql_exception: Table 'db_yunes.cabecera' doesn't exist in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\CartController.php(118): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\CartController->confirmarCompra('')
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CartController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 09:00:20 --> Table 'db_yunes.cabecera' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (14000, \'2023-06-13 09:00:20   \', \'5\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (14000, \'2023-06-13 09:00:20   \', \'5\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (14000, \'2023-06-13 09:00:20   \', \'5\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (:total_venta:, :fecha:, :usuario_id:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\CartController.php(118): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\CartController->confirmarCompra('')
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CartController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 09:22:02 --> mysqli_sql_exception: Table 'db_yunes.cabecera' doesn't exist in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\CartController.php(118): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\CartController->confirmarCompra('')
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CartController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 09:22:02 --> Table 'db_yunes.cabecera' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (14000, \'2023-06-13 09:22:02   \', \'5\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (14000, \'2023-06-13 09:22:02   \', \'5\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (14000, \'2023-06-13 09:22:02   \', \'5\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (:total_venta:, :fecha:, :usuario_id:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\CartController.php(118): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\CartController->confirmarCompra('')
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CartController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 10:21:59 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`bd_yunesnicolas`.`detalle`, CONSTRAINT `detalle_ibfk_1` FOREIGN KEY (`producto_id`) REFERENCES `productos` (`id`)) in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `de...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `de...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `de...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `de...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\CartController.php(128): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\CartController->confirmarCompra('')
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CartController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 10:21:59 --> Cannot add or update a child row: a foreign key constraint fails (`bd_yunesnicolas`.`detalle`, CONSTRAINT `detalle_ibfk_1` FOREIGN KEY (`producto_id`) REFERENCES `productos` (`id`))
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `detalle` (`venta_id`, `producto_id`, `cantidad`, `precio`) VALUES (39, \'18\', 3, 7000)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `detalle` (`venta_id`, `producto_id`, `cantidad`, `precio`) VALUES (39, \'18\', 3, 7000)')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `detalle` (`venta_id`, `producto_id`, `cantidad`, `precio`) VALUES (39, \'18\', 3, 7000)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `detalle` (`venta_id`, `producto_id`, `cantidad`, `precio`) VALUES (:venta_id:, :producto_id:, :cantidad:, :precio:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\CartController.php(128): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\CartController->confirmarCompra('')
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CartController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 10:22:19 --> mysqli_sql_exception: Table 'db_yunes.cabecera' doesn't exist in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\CartController.php(118): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\CartController->confirmarCompra('')
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CartController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 10:22:19 --> Table 'db_yunes.cabecera' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (21000, \'2023-06-13 10:22:19   \', \'5\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (21000, \'2023-06-13 10:22:19   \', \'5\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (21000, \'2023-06-13 10:22:19   \', \'5\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (:total_venta:, :fecha:, :usuario_id:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\CartController.php(118): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\CartController->confirmarCompra('')
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CartController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 10:22:44 --> mysqli_sql_exception: Table 'db_yunes_nicolas.detalle' doesn't exist in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `de...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `de...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `de...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `de...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\CartController.php(128): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\CartController->confirmarCompra('')
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CartController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 10:22:44 --> Table 'db_yunes_nicolas.detalle' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `detalle` (`venta_id`, `producto_id`, `cantidad`, `precio`) VALUES (39, \'18\', 3, 7000)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `detalle` (`venta_id`, `producto_id`, `cantidad`, `precio`) VALUES (39, \'18\', 3, 7000)')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `detalle` (`venta_id`, `producto_id`, `cantidad`, `precio`) VALUES (39, \'18\', 3, 7000)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `detalle` (`venta_id`, `producto_id`, `cantidad`, `precio`) VALUES (:venta_id:, :producto_id:, :cantidad:, :precio:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\CartController.php(128): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\CartController->confirmarCompra('')
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CartController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 10:49:51 --> mysqli_sql_exception: Table 'db_yunes.cabecera' doesn't exist in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\CartController.php(118): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\CartController->confirmarCompra('')
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CartController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 10:49:51 --> Table 'db_yunes.cabecera' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (21000, \'2023-06-13 10:49:51   \', \'5\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (21000, \'2023-06-13 10:49:51   \', \'5\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (21000, \'2023-06-13 10:49:51   \', \'5\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (:total_venta:, :fecha:, :usuario_id:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\CartController.php(118): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\CartController->confirmarCompra('')
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CartController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 10:54:07 --> mysqli_sql_exception: Table 'db_yunes.cabecera' doesn't exist in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\Yunes\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\Yunes\app\Controllers\CartController.php(118): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\CartController->confirmarCompra('')
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CartController))
#10 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-13 10:54:07 --> Table 'db_yunes.cabecera' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (7000, \'2023-06-13 10:54:07   \', \'5\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (7000, \'2023-06-13 10:54:07   \', \'5\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (7000, \'2023-06-13 10:54:07   \', \'5\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `cabecera` (`total_venta`, `fecha`, `usuario_id`) VALUES (:total_venta:, :fecha:, :usuario_id:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\CartController.php(118): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\CartController->confirmarCompra('')
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\CartController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 14:19:35 --> The original file is not a valid file.
in SYSTEMPATH\HTTP\Files\UploadedFile.php on line 137.
 1 SYSTEMPATH\HTTP\Files\UploadedFile.php(137): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidFile()
 2 APPPATH\Controllers\ProductController.php(124): CodeIgniter\HTTP\Files\UploadedFile->move('assets/img/productos/', '1686665975_5e3d08cf23e6b53e845e')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProductController->actualizar()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 14:21:47 --> unlink(assets/img/productos/1686665049_07e243e89b5f902be994.png): No such file or directory
in APPPATH\Controllers\ProductController.php on line 120.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(assets/img/productos/1686665049_07e243e89b5f902be994.png): No such file or directory', 'C:\\xampp\\htdocs\\Yunes\\app\\Controllers\\ProductController.php', 120)
 2 APPPATH\Controllers\ProductController.php(120): unlink('assets/img/productos/1686665049_07e243e89b5f902be994.png')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProductController->actualizar()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 14:28:50 --> unlink(assets/img/productos/1686665049_07e243e89b5f902be994.png): No such file or directory
in APPPATH\Controllers\ProductController.php on line 120.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(assets/img/productos/1686665049_07e243e89b5f902be994.png): No such file or directory', 'C:\\xampp\\htdocs\\Yunes\\app\\Controllers\\ProductController.php', 120)
 2 APPPATH\Controllers\ProductController.php(120): unlink('assets/img/productos/1686665049_07e243e89b5f902be994.png')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProductController->actualizar()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 14:29:20 --> unlink(assets/img/productos/1686665049_07e243e89b5f902be994.png): No such file or directory
in APPPATH\Controllers\ProductController.php on line 120.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(assets/img/productos/1686665049_07e243e89b5f902be994.png): No such file or directory', 'C:\\xampp\\htdocs\\Yunes\\app\\Controllers\\ProductController.php', 120)
 2 APPPATH\Controllers\ProductController.php(120): unlink('assets/img/productos/1686665049_07e243e89b5f902be994.png')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProductController->actualizar()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 14:38:06 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 924.
 1 SYSTEMPATH\BaseModel.php(924): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\ConsultasController.php(90): CodeIgniter\Model->update('1', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->borrar('1')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 14:38:47 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 924.
 1 SYSTEMPATH\BaseModel.php(924): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\ConsultasController.php(90): CodeIgniter\Model->update('1', [...])
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->borrarConsulta('1')
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 14:46:57 --> Undefined variable $item
in APPPATH\Views\consultas.php on line 30.
 1 APPPATH\Views\consultas.php(30): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $item', 'C:\\xampp\\htdocs\\Yunes\\app\\Views\\consultas.php', 30)
 2 SYSTEMPATH\View\View.php(213): include('C:\\xampp\\htdocs\\Yunes\\app\\Views\\consultas.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('consultas', [], true)
 5 APPPATH\Controllers\ConsultasController.php(22): view('consultas', [...])
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ConsultasController->cargarConsultas()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ConsultasController))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 14:52:40 --> mysqli_sql_exception: Table 'db_yunes.cabecera' doesn't exist in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\Yunes\app\Controllers\VentasController.php(31): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-06-13 14:52:40 --> Table 'db_yunes.cabecera' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `cabecera`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `cabecera`')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `cabecera`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `cabecera`', [], false)
 5 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\VentasController.php(31): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 15:05:24 --> mysqli_sql_exception: Table 'db_yunes.detalle' doesn't exist in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\Yunes\app\Controllers\VentasController.php(32): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-06-13 15:05:24 --> Table 'db_yunes.detalle' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `detalle`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `detalle`')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `detalle`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `detalle`', [], false)
 5 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\VentasController.php(32): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-13 15:22:34 --> mysqli_sql_exception: Table 'db_yunes.detalle' doesn't exist in C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Yunes\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\Yunes\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\Yunes\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Yunes\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Yunes\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\Yunes\app\Controllers\VentasController.php(32): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
#8 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
#9 C:\xampp\htdocs\Yunes\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Yunes\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-06-13 15:22:34 --> Table 'db_yunes.detalle' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *
FROM `detalle`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `detalle`')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `detalle`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `detalle`', [], false)
 5 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\VentasController.php(32): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\VentasController->ventasRealizadas()
 9 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\VentasController))
10 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 18:24:01 --> The original file is not a valid file.
in SYSTEMPATH\HTTP\Files\UploadedFile.php on line 137.
 1 SYSTEMPATH\HTTP\Files\UploadedFile.php(137): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidFile()
 2 APPPATH\Controllers\ProductController.php(125): CodeIgniter\HTTP\Files\UploadedFile->move('assets/img/productos/', '1686680641_ec81ac84b63ffc7a014c')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProductController->actualizar()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 18:24:45 --> unlink(assets/img/productos/1686657290_4eefe0b26a12d5c9cbcf.png): No such file or directory
in APPPATH\Controllers\ProductController.php on line 121.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(assets/img/productos/1686657290_4eefe0b26a12d5c9cbcf.png): No such file or directory', 'C:\\xampp\\htdocs\\Yunes\\app\\Controllers\\ProductController.php', 121)
 2 APPPATH\Controllers\ProductController.php(121): unlink('assets/img/productos/1686657290_4eefe0b26a12d5c9cbcf.png')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProductController->actualizar()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 18:24:55 --> unlink(assets/img/productos/1686657290_4eefe0b26a12d5c9cbcf.png): No such file or directory
in APPPATH\Controllers\ProductController.php on line 121.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(assets/img/productos/1686657290_4eefe0b26a12d5c9cbcf.png): No such file or directory', 'C:\\xampp\\htdocs\\Yunes\\app\\Controllers\\ProductController.php', 121)
 2 APPPATH\Controllers\ProductController.php(121): unlink('assets/img/productos/1686657290_4eefe0b26a12d5c9cbcf.png')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProductController->actualizar()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 18:26:31 --> unlink(assets/img/productos/1686657290_4eefe0b26a12d5c9cbcf.png): No such file or directory
in APPPATH\Controllers\ProductController.php on line 121.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(assets/img/productos/1686657290_4eefe0b26a12d5c9cbcf.png): No such file or directory', 'C:\\xampp\\htdocs\\Yunes\\app\\Controllers\\ProductController.php', 121)
 2 APPPATH\Controllers\ProductController.php(121): unlink('assets/img/productos/1686657290_4eefe0b26a12d5c9cbcf.png')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProductController->actualizar()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 18:26:36 --> unlink(assets/img/productos/1686657290_4eefe0b26a12d5c9cbcf.png): No such file or directory
in APPPATH\Controllers\ProductController.php on line 121.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(assets/img/productos/1686657290_4eefe0b26a12d5c9cbcf.png): No such file or directory', 'C:\\xampp\\htdocs\\Yunes\\app\\Controllers\\ProductController.php', 121)
 2 APPPATH\Controllers\ProductController.php(121): unlink('assets/img/productos/1686657290_4eefe0b26a12d5c9cbcf.png')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProductController->actualizar()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-13 18:28:52 --> unlink(assets/img/productos/1686657290_4eefe0b26a12d5c9cbcf.png): No such file or directory
in APPPATH\Controllers\ProductController.php on line 121.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'unlink(assets/img/productos/1686657290_4eefe0b26a12d5c9cbcf.png): No such file or directory', 'C:\\xampp\\htdocs\\Yunes\\app\\Controllers\\ProductController.php', 121)
 2 APPPATH\Controllers\ProductController.php(121): unlink('assets/img/productos/1686657290_4eefe0b26a12d5c9cbcf.png')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\ProductController->actualizar()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
